.github/workflows/linting.yaml

```diff
@@ -40,6 +43,9 @@ jobs:
       ) &&
       contains(github.event.pull_request.labels.*.name, vars.LBL_MRKDWNPROSE)
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        site: ['platform', 'upsun']
     steps:
       - name: Checkout
         uses: actions/checkout@v3
```

Comment:
we now need to add 
`ref: ${{ github.event.pull_request.head.ref }}` 
to the checkout action so we can pull the code from the PR, not from the target. 

To limit malicious injections, we're only running this job if the `build-fork` label has been added so we need to extra cautious when applying that label to PRs from forks.